DigitalLearningSolutions.Web/ViewModels/LearningMenu/TutorialViewModel.cs

```diff
@@ -1,23 +1,32 @@
 ﻿namespace DigitalLearningSolutions.Web.ViewModels.LearningMenu
 {
-    using System.Linq;
-    using System.Net;
     using DigitalLearningSolutions.Data.Models.TutorialContent;
+    using DigitalLearningSolutions.Web.Helpers;
     using HtmlAgilityPack;
+    using Microsoft.Extensions.Configuration;
 
     public class TutorialViewModel
     {
         public TutorialInformation TutorialInformation { get; }
         public string? Objectives { get; }
         public int CustomisationId { get; }
         public int SectionId { get; }
+        public string? SupportingMaterialPath { get; }

```

Comment:
Having one supporting material path here and one inside tutorial information I think is a bit confusing and could easily lead to mistakes. As soon as we're doing any parsing of the tutorial information data I think we should store the fields we need in this class instead of the whole tutorial information object 🙂

Reply:
Yeah I had forgotten about that, I've updated this in ca246da. It's using a mix of formatting some fields at construction, and others on request. Is this ok, or should I change it so all fields are parsed/formatted when the view model is being constructed? 🙂

Reply:
I'd recommend changing it so all fields are parsed/formatted when the view model is being constructed 🙂

Reply:
Done in 719b12c 🙂